### DISCUSSION TITLE
Filter by Title

### DISCUSSION THREAD

**Discussion by volitanic** [jmbannon/ytdl-sub#749] [Q&A] [answered]
I am having no luck with filtering by title.  

My goal is to only grab files with FP1, FP2 or FP3 in the titles from here: https://www.youtube.com/channel/UCB_qr75-ydFVKSF9Dmo6izg.  How would I go about doing this?  I've spent about 3 hours with the docs and trying things without luck. 

Thanks!

**Comment by jmbannon** [comment]
Either match-filters (https://ytdl-sub.readthedocs.io/en/latest/config.html#match-filters) or regex (https://ytdl-sub.readthedocs.io/en/latest/config.html#regex)

Would go with match-filters

**Comment by jmbannon** [comment, marked as answer]
From yt-dlp docs:
https://github.com/yt-dlp/yt-dlp#filtering-formats

You can add in your preset/subscription:

```
match_filters:
  filters:
    - "title*=FP1"
    - "title*=FP2"
    - "title*=FP3"
```
The video will download if one of them matches. `*=` means 'contains'
